content/300-guides/200-deployment/110-deployment-guides/150-deploying-to-azure-functions.mdx

```diff
@@ -187,18 +187,18 @@ curl ifconfig.me
 Copy the IP from the output and run the following command, replacing `YOUR_PUBLIC_IP` with the IP address and `UNIQUE_DB_SERVER_NAME` with the name from step 4:
 
 ```terminal copy
-az sql server firewall-rule create --resource-group prisma-azure-example --server UNIQUE_DB_SERVER_NAME --name allow-local-acccess --start-ip-address YOUR_PUBLIC_IP --end-ip-address YOUR_PUBLIC_IP
+az sql server firewall-rule create --resource-group prisma-azure-example --server UNIQUE_DB_SERVER_NAME --name allow-local-access --start-ip-address YOUR_PUBLIC_IP --end-ip-address YOUR_PUBLIC_IP
```

Comment:
Ohoh, this also made it into https://devblogs.microsoft.com/azure-sql/modern-application-development-with-prisma-graphql-rest-and-azure-sql/
@nikolasburk @ruheni

Reply:
Alex can you reach out to Davide to get this updated?

Reply:
I've reached out to Davide.

Reply:
Done!